Sources/Animations/KeyboardConstraint.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+/// Set any NSLayoutConstraint class to this class and it moves up and down with the keyboard.
+open class KeyboardConstraint: NSLayoutConstraint {
+    /// this amount is added to the keyboard height to show the view above the keyboard.
+    public var offsetFromKeyboardHeight: CGFloat = 15
+    /// will be set on awakeFromNib to the constant value. 
+    /// Is used to move back to the position before the keyboard was shown
+    public var normalConstant: CGFloat = 0
```

Comment:
Nee, en ja . Die wordt wel gezet vanaf storyboard ja. Dus kan in princiepe private... zal er is over denken.